certs/dodcerts.sh

```diff
@@ -0,0 +1,5 @@
+yum install -y unzip
+url='https://public.cyber.mil/pki-pke/pkipke-document-library/'
+bundle=$(curl $url | grep -ioP '(?<=").*dod.zip(?=")')
+curl -o dodcerts.zip "$bundle"
+unzip -l dodcerts.zip | grep -ioP '.*\s+\K.*.pem' | xargs -I{} unzip -o -j dodcerts.zip {} -d /etc/pki/ca-trust/source/anchors/
```

Comment:
refer back to the gist to be able to handle crt files as well, not just pem files

Reply:
Turned out that the PEM file only contained one cert for use for validating correctness.  Switched implementation to use the DER file chain.